src/resources/hosts/llnl/host_llnl_rztopaz.xml

```diff
@@ -1,10 +1,9 @@
 <?xml version="1.0"?>
```

Comment:
can we just delete this file? rztopaz retires april 2nd.

Reply:
I went back and forth on this, I think we can remove when the system is truly gone.